src/components/OrderableList/OrderableList.tsx

```diff
@@ -0,0 +1,259 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+import { useDroppableCollection } from "@react-aria/dnd";
+import { useGrid } from "@react-aria/grid";
+import { ListKeyboardDelegate } from "@react-aria/selection";
+import { mergeProps } from "@react-aria/utils";
+import { Item } from "@react-stately/collections";
+import { useDraggableCollectionState, useDroppableCollectionState } from "@react-stately/dnd";
+import { GridCollection, useGridState } from "@react-stately/grid";
+import { useListState } from "@react-stately/list";
+import { DropTarget, ItemDropTarget } from "@react-types/shared";
+import React, { useRef } from "react";
+import { GridNode } from "@react-types/grid";
+import { InsertionIndicator } from "./InsertionIndicator";
+import { CollectionItem } from "./CollectionItem";
+import { ItemDragState, OrderableListContainerProps, OrderableListItem, OrderableListProps } from "./types";
+import { useMemoizedId } from "@hooks/useMemoizedId";
+
+export const OrderableList = <T extends object>({
+    onMove,
+    items,
+    renderContent,
+    dragDisabled,
+    disableTypeAhead,
+}: OrderableListProps<T>) => (
+    <OrderableListContainer
+        onMove={onMove}
+        renderContent={renderContent}
+        items={items}
+        disableTypeAhead={disableTypeAhead}
+        dragDisabled={dragDisabled}
+    >
+        {(item: OrderableListItem<T>) => (
+            <Item textValue={item.alt}>
+                <></>
```

Comment:
Is this needed?

Reply:
For React-aria, yes. The list state requires the child prop that is passed to the orderablelistcontainer to be a function which returns an item. It is possible to manually add this to the props inside the orderablelistcontainer though as an alternative

Reply:
No, that's fine :) but maybe add a comment that explains why there has to be an empty child element

Reply:
done :)